source/reference/method/MongoDBClient-bulkWrite.txt

```diff
@@ -0,0 +1,100 @@
+============================
+MongoDB\\Client::bulkWrite()
+============================
+
+.. meta::
+   :description: Perform write operations across multiple namespaces by using the MongoDB PHP Library.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. versionadded:: 2.1
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\Client::bulkWrite()
+
+   Executes multiple write operations across multiple namespaces.
+
+   .. code-block:: php
+
+      function bulkWrite(
+          BulkWriteCommand|ClientBulkWrite $bulk,
+          array $options = []
+      ): MongoDB\BulkWriteCommandResult
+
+Parameters
+----------
+
+``$bulk`` : :phpclass:`MongoDB\ClientBulkWrite` or ``BulkWriteCommand``
+  (reserve for use with the {+extension-short+} only)
```

Comment:
`MongoDB\Client` is provided by the library, "extension-only" makes no sense if you call a library method.
I think the intention is to be compatible with the driver object because this might be useful and this was not complex to support.

Reply:
I can remove this parentheses and make a note instead